### ISSUE TITLE
Add ability to disable log exception

### ISSUE THREAD

**Issue by dengber** [csnemes/tracer#78]
The "try{...}catch(Exception e){log(e)}" waved into code cause wrong line number of exception stack trace.

Please refer the flowing URL for detailed reason.
https://stackoverflow.com/questions/28481577/postsharps-onexceptionaspect-is-corrupting-the-stack-traces-line-numbers/28541633
https://support.postsharp.net/request/21942-exception-line-number-at-end-of

Can you add the ability to disable log exception when user prefer to disable it?

**Comment by csnemes** [comment]
Yes of course, I'll put it into the next iteration. Also I'll try to see if using the ExceptioDispatchInfo class can fix this problem.

**Comment by dengber** [comment]
Thanks!
